git-duet/main.go

```diff
@@ -109,11 +116,11 @@ func printAuthor(author *duet.Pair) {
 	fmt.Printf("GIT_AUTHOR_EMAIL='%s'\n", author.Email)
 }
 
-func printCommitter(committer *duet.Pair) {
-	if committer == nil {
+func printNextComitter(committers []*duet.Pair) {
+	if committers == nil {
 		return
 	}
 
-	fmt.Printf("GIT_COMMITTER_NAME='%s'\n", committer.Name)
-	fmt.Printf("GIT_COMMITTER_EMAIL='%s'\n", committer.Email)
+	fmt.Printf("GIT_COMMITTER_NAME='%s'\n", committers[0].Name)
```

Comment:
It should theoretically never happen, but it might be nice to return early if `committers` is not `nil`, but `len(committers) == 0`.